docs/getting_started/basic_walkthrough.adoc

```diff
@@ -67,14 +77,25 @@ Variables will be URL-encoded when they are injected into the URL.
 .Create action URLs using parameters.
 image::gf-custom-actions.png[Custom Actions, 500]
 
+To create a custom action, follow these steps:
+
+. Navigate to the http://127.0.0.1:3000/[home page] of your Grafana instance.
+. In the top-left corner of the page, click on the Grafana icon, and then click *Plugins*.
+. Select OpenNMS Helm. 
+. On the Confg tab, click *+Add Custom Action*:
+
+image::gf-add-cust-action.png[Custom Action, 500]
+
+. Specify a label and add  
```

Comment:
Specify a label and `click` add?

Reply:
This is a result of being new to git. I am not finished that file, and I had thought I had excluded it from my pull, but I didn't. I'll try to fix this.